scripts/doneTask.js

```diff
@@ -0,0 +1,11 @@
+import { renderTodoList } from "../index.js";
+import { todos } from "./deteleTask.js";
+
+export const handleDone = (taskId, doneState) => {
+  const task = todos.find((todo) => todo.id === taskId);
```

Comment:
This is a common pitfall many FE dev fall in.
`Array.prototype.find` returns undefined when it doesn't find anything.
So we have to careful about it as if we try to do `undefined.someValue` it's going to crash our app.
```suggestion
  const task = todos.find((todo) => todo.id === taskId);
  if(task === undefined){
      return
  }
```